model-optimizer/extensions/front/tf/RetinaNetFilteredDetectionsReplacement.py

```diff
@@ -246,9 +246,17 @@ def generate_sub_graph(self, graph: Graph, match: SubgraphMatch):
                                                                    applied_width_height_regressions_node)
 
         detection_output_op = DetectionOutput(graph, match.custom_replacement_desc.custom_attributes)
+        # get nms from original network
+        for node_name in match.matched_nodes_names():
+            if Node(graph, node_name)['op'] == 'NonMaxSuppression':
```

Comment:
```
node = Node(graph, node_name)
if node.soft_get('op') == 'NonMaxSuppression':
    if node.is_in_port_connected(3):
        iou_threshold = Node(graph, node_name).in_node(3).value
    else:
        iou_threshold = 0.0 # This is the default value for the TF NMS, right?
    break
```

Reply:
In TF it's not defined explicitly https://www.tensorflow.org/api_docs/python/tf/raw_ops/NonMaxSuppressionV3 But conventionally default value is 0.5. But I would prefer not set it here by hand since it must've been defined in the original network

Reply:
Agree